### PR TITLE
Feature/cdpt 1751 search bar

### PR DESCRIPTION
**Pull request by EmilyHazlehurst**

### CODE REVIEW

File: public/app/frontend/src/components/search-bar/search-bar.html.twig [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    {% include '@components/search-bar/search-bar.html.twig' with {
+        action: '/',
+        id: 'search-top'
+        input: {
+            labelHidden: true,
+            label: 'Search',
+            id: 'search-top-input',
+        },
+        button: {
+            style: 'primary',
```
Comment (EarthlingDavey) [inline public/app/frontend/src/components/search-bar/search-bar.html.twig:20]: Super minor, as style is sometimes used in jsx as css. e.g. `style={{'padding':'10px'}}`.

Could we use `variant: 'primary'` or something else other than style?

File: public/app/frontend/src/components/navigation-main/navigation-main.scss [hunk 2]
```diff
@@ -13,6 +13,7 @@
         width: 100%;
         @media screen and (min-width: $justice-lg) {
             flex-direction: row;
+            margin-left: -0.625rem;
```
Comment (EarthlingDavey) [inline public/app/frontend/src/components/navigation-main/navigation-main.scss:16]: Could this be replaced with a new variable, or a a fraction of an existing variable like `$gutter / 2`?

**Review by EarthlingDavey — APPROVED** [review]

File: public/app/frontend/src/components/search-bar/search-bar.html.twig (hunk 1 above)
Comment (EmilyHazlehurst) [reply public/app/frontend/src/components/search-bar/search-bar.html.twig:20]: Good shout! I've updated that :)

File: public/app/frontend/src/components/navigation-main/navigation-main.scss (hunk 2 above)
Comment (EmilyHazlehurst) [reply public/app/frontend/src/components/navigation-main/navigation-main.scss]: Done :)

**Pull request merged**